tests/registry/main.c

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright (C) 2018 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @file
+ * @brief       Shows the use of the RIOT registry for handling runtime
+ *              configurations using multiple store options.
+ *
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include "errno.h"
+#include "shell.h"
+#include "board.h"
+#include "registry/registry.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#ifdef MODULE_REGISTRY_STORE_DUMMY
+#include "dummy/include/registry_store_dummy.h"
+registry_dummy_t registry_dummy_storage;
+#endif /* MODULE_REGISTRY_STORE_DUMMY */
+
+/* Size of the test_bytes configuration parameter */
+#ifndef BYTES_LENGTH
+#define BYTES_LENGTH    16
+#endif
+
+/* These are the 'configuration parameters' */
+int64_t test_opt1 = 0;
+int8_t test_opt2 = 1;
+unsigned char test_bytes[BYTES_LENGTH] = {0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA,
+                                          0xAA, 0xAA, 0xAA, 0xAA, 0xAA, 0xAA,
+                                          0xAA, 0xAA, 0xAA, 0xAA};
+
```

Comment:
it's base64

Reply:
I guess this is chosen because base64 makes the encoded data less likely to be modified in transit? i.e. through the configuration management system (serial/lwm2m/etc)?